WorkflowReactiveSwift/Tests/WorkerTests.swift

```diff
@@ -64,23 +64,26 @@ class WorkerTests: XCTestCase {
     // A worker declared on a first `render` pass that is not on a subsequent should have the work cancelled.
     func test_cancelsWorkers() {
         struct WorkerWorkflow: Workflow {
-            var startExpectation: XCTestExpectation
-            var endExpectation: XCTestExpectation
-
             enum State {
                 case notWorking
-                case working
+                case working(start: XCTestExpectation, end: XCTestExpectation)
+            }
+
+            let initialState: State
+
+            func makeInitialState() -> State {
+                initialState
             }
 
-            func makeInitialState() -> WorkerWorkflow.State {
-                return .notWorking
+            func workflowDidChange(from previousWorkflow: WorkerWorkflow, state: inout State) {
+                state = initialState
```

Comment:
Any reason to use `State` here at all? (rather than just having that enum be a property on the workflow and using `Void` for state?)

Reply:
Oh god! :) My brain's totally fried. 

Of course, I'll make that change.

Reply:
Hah, it happens 👍